cluster/gce/gci/configure-helper.sh

```diff
@@ -2349,6 +2349,21 @@ EOF
   fi
 }
 
+# Sets up the manifests of local dns cache agent for k8s addons.
+function setup-nodelocaldns-manifest {
+  local -r localdns_file="${dst_dir}/dns/nodelocaldns/nodelocaldns.yaml"
+  mv "${dst_dir}/dns/nodelocaldns/nodelocaldns.yaml.in" "${localdns_file}"
+  # Replace the salt configurations with variable values.
+  sed -i -e "s@{{ *pillar\['dns_domain'\] *}}@${DNS_DOMAIN}@g" "${localdns_file}"
+  sed -i -e "s@{{ *pillar\['dns_server'\] *}}@${DNS_SERVER_IP}@g" "${localdns_file}"
+  sed -i -e "s@{{ *pillar\['local_dns_ip'\] *}}@${LOCAL_DNS_IP}@g" "${localdns_file}"
+  if [[ -n "${CLUSTER_IP_RANGE:-}" ]]; then
+    sed -i -e "s@{{ *pillar\['service_cluster_ip_range'\] *}}@${CLUSTER_IP_RANGE}@g" "${localdns_file}"
```

Comment:
I don't see this variable anywhere in the yaml files, so I'm not sure what the purpose of this is.

Reply:
I added this to use as access control for dns requests. It was an option for Unbound config file, but since we are using CoreDNS this is currently unused. I will remove it.